### PR TITLE
Add sensu-backend init in installation and backend reference

### LINKED ISSUE #1917: Document sensu-backend init
Proposal: https://docs.google.com/document/d/14exgupRiOp1DBieDs51Nshl9E8mfKETbvvOWm4tG2Mk/edit

## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [x] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Context
Create a guide to document usage around `sensu-backend init`. Rather than providing the default `admin` user with `P@ssw0rd!`, this command must be run on _new_ Sensu installations prior to use. See https://github.com/sensu/sensu-go/issues/3359 for implementation.

Requires https://github.com/sensu/sensu-go/issues/3359.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Add initialization instructions in installation steps and backend reference.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1917

### CODE REVIEW

**Comment by hillaryfraley** [comment]
@nikkixdev I would like to try this out myself if there's a way to do it. Cameron suggested maybe a Circle CI build that I could run in a skunkworks machine?

File: content/sensu-go/5.16/installation/install-sensu.md [hunk 1]
```diff
@@ -77,7 +77,19 @@ sudo yum install sensu-go-backend
 
 {{< /language-toggle >}}
 
-##### 2. Configure and start
+##### 2. Initialize
+
+Run `sensu-backend init` to set up your Sensu administrator username and password. In this initialization step, you only need to provide a username and password string &mdash; no need for role-based access control (RBAC) or etcd.
```
Comment (echlebek) [inline content/sensu-go/5.16/installation/install-sensu.md]: > no need for role-based access control (RBAC) or etcd.

The tool connects to etcd, using the configuration supplied in backend.yml, so it's probably best not to mention etcd in this context

File: content/sensu-go/5.16/installation/install-sensu.md [hunk 2]
```diff
@@ -77,7 +77,19 @@ sudo yum install sensu-go-backend
...
+
+Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use:
+
+{{< highlight shell >}}
+sensu-backend init --admin-username YOUR_USERNAME --admin-password YOUR_PASSWORD
+{{< /highlight >}}
```
Comment (echlebek) [inline content/sensu-go/5.16/installation/install-sensu.md]: While this is a supported mechanism of the tool, it is also insecure. The tool's parameters will be visible to other users in the system.

A better approach is to set environment variables.

```
{{< highlight shell >}}
export SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=YOUR_USERNAME
export SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=YOUR_PASSWORD
sensu-backend init
{{< /highlight >}}

File: content/sensu-go/5.16/reference/backend.md [hunk 3]
```diff
@@ -57,6 +58,33 @@ For information about creating and managing checks, see:
 - [Guide to collecting metrics with checks][4]
 - [Checks reference documentation][5]
 
+## Initialization
+
+Before you start the Sensu backend on a **new** installation, you must set up an administrator username and password. Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use:
+
+{{< highlight shell >}}
+sensu-backend init --admin-username YOUR_USERNAME --admin-password YOUR_PASSWORD
```
Comment (echlebek) [inline content/sensu-go/5.16/reference/backend.md]: Let's nudge people towards setting environment variables here as well

**Review by echlebek — COMMENTED** [review]
Looks good in general, but I'd like to discourage people from using the flags method. Even though it works, it can leak secrets to other users of the operating system, so it's not a good practice.

**Review by echlebek — APPROVED** [review]
:shipit: 

```
              |    |    |
             )_)  )_)  )_)
            )___))___))___)\
           )____)____)_____)\\
         _____|____|____|____\\\__
---------\                   /---------
  ^^^^^ ^^^^^^^^^^^^^^^^^^^^^
    ^^^^      ^^^^     ^^^    ^^
         ^^^^      ^^^
```

**Pull request merged**